docker-compose.yml

```diff
@@ -1,133 +1,102 @@
   web:
-    build: .
-    image: ghcr.io/mastodon/mastodon:v4.2.11
-    restart: always
+    # Remove the comments from this `build` section if you need to build the image from source.
+    # build:
+    #   context: .
+    #   dockerfile: Dockerfile
+    image: ghcr.io/mastodon/mastodon:*image-tag
     env_file: .env.production
+    environment:
+      - PORT=3000
+      - BIND=127.0.0.1
+    restart: always
     command: bundle exec puma -C config/puma.rb
+    healthcheck:
+      test:
+        [
+          'CMD-SHELL',
+          "curl -s --noproxy localhost localhost:3000/health | grep -q 'OK' || exit 1",
+        ]
     networks:
       - external_network
       - internal_network
-    healthcheck:
-      # prettier-ignore
-      test: ['CMD-SHELL', 'wget -q --spider --proxy=off localhost:3000/health || exit 1']
     ports:
       - '127.0.0.1:3000:3000'
-    depends_on:
-      - db
-      - redis
-      # - es
+    # S3 is optional but recommended for storing user-uploaded media files.
     volumes:
       - ./public/system:/mastodon/public/system
 
+  # This is the Mastodon streaming API. It serves the streaming API used by the web interface.
   streaming:
-    build: .
-    image: ghcr.io/mastodon/mastodon:v4.2.11
-    restart: always
+    # Remove the comments from this `build` section if you need to build the image from source.
+    # build:
+    #   context: ./streaming
+    #   dockerfile: Dockerfile
+    image: ghcr.io/mastodon/mastodon-streaming:*image-tag
     env_file: .env.production
-    command: node ./streaming
+    environment:
+      - PORT=4000
+      - BIND=127.0.0.1
+    restart: always
+    command: node ./streaming/index.js
+    healthcheck:
+      test:
+        [
+          'CMD-SHELL',
+          "curl -s --noproxy localhost localhost:4000/api/v1/streaming/health | grep -q 'OK' || exit 1",
+        ]
     networks:
       - external_network
       - internal_network
-    healthcheck:
-      # prettier-ignore
-      test: ['CMD-SHELL', 'wget -q --spider --proxy=off localhost:4000/api/v1/streaming/health || exit 1']
     ports:
       - '127.0.0.1:4000:4000'
-    depends_on:
-      - db
-      - redis
 
-  sidekiq:
-    build: .
-    image: ghcr.io/mastodon/mastodon:v4.2.11
-    restart: always
-    env_file: .env.production
-    command: bundle exec sidekiq
-    depends_on:
-      - db
-      - redis
-    networks:
-      - external_network
-      - internal_network
-    volumes:
-      - ./public/system:/mastodon/public/system
-    healthcheck:
-      test: ['CMD-SHELL', "ps aux | grep '[s]idekiq\ 6' || false"]
+  # You can run multiple Sidekiq processes to handle more jobs concurrently. The scheduler queue must only be run on one process.
+  # Please see https://docs.joinmastodon.org/admin/scaling/#sidekiq for more information.
+  sidekiq-1:
+    <<: *sidekiq-base
+    command: bundle exec sidekiq -c 15 -q default -q push -q ingress -q mailers -q pull -q scheduler
+
+  sidekiq-2:
```

Comment:
I would not enable 2 sidekiq containers by default, but maybe comment this part?